lib/mutant/ast/regexp/transformer/root.rb

```diff
@@ -0,0 +1,27 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        class Root < self
+          register :regexp_root_expression
+
+          class ExpressionToAST < Transformer::ExpressionToAST
+            def call
+              quantify(ast(*children))
+            end
+          end
+
+          class ASTToExpression < Transformer::ASTToExpression
+            def call
+              root = ::Regexp::Expression::Root.new
+
```

Comment:
Instance of https://github.com/mbj/mutant/pull/565/commits/b3b6c22e8868baee9691c75127e3baba833ce8c7#r61765189